### DISCUSSION TITLE
Industrial(ish) AC control

### DISCUSSION THREAD

**Discussion by dac2020** [arendst/Tasmota#16521] [Q&A] [answered]
I look at the list of supported hardware but I see mostly home hardware.
Do you know of any reasonably priced plug controls that are a bit higher quality than home hardware?
I'm looking for almost industrial but without the industrial price :).

Figured I'd ask here in case someone knows of some devices.

Thank you.

**Comment by barbudor** [comment, marked as answer]
Wifi plugs are almost by definition a home device intended to be used by a consummer.

For industrial-ish, I would look at Din rail mountable device such as those already listed in https://github.com/arendst/Tasmota/discussions/16333#discussioncomment-3478264
Including also ESP32R4 and Rocketcontroller/ASTRA listed in https://github.com/arendst/Tasmota/discussions/16394

You can add the Shelly Pro family: https://shelly.cloud/shelly-pro-smart-home-automation-solution/

If that's not enough industrial for you, you will have to build it yourself

**Reply by dac2020** [reply]
That's great. I had forgotten about that thread so thanks for showing it again.
